src/main/java/com/squareup/javapoet/AnnotationSpec.java

```diff
@@ -226,6 +216,27 @@ public Builder addMember(String name, CodeBlock codeBlock) {
       return this;
     }
 
+    /**
+     * Delegates to {@link #addMember(String, String, Object...)}, with parameter {@code format}
+     * depending on the given {@code value} object. Falls back to {@code "$L"} literal format if
+     * the class of the given {@code value} object is not supported.
+     */
+    private Builder addValue(String memberName, Object value) {
```

Comment:
*ganrflrrrg* ...then we're back to the naming conflict discovered by you. See #365 :smile: